### PR TITLE
chore: Remove CodeCov from CI

### PR DESCRIPTION
**Pull request by matmerr**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: .pipelines/singletenancy/aks-swift/e2e-job-template.yaml [hunk 1]
```diff
@@ -18,7 +18,6 @@ stages:
           - Role -equals $(CUSTOM_E2E_ROLE)
         container:
           image: "$(BUILD_IMAGE)" # build image set as variable in pipeline runtime for flexibilit
-
```
Comment (pjohnst5) [inline .pipelines/singletenancy/aks-swift/e2e-job-template.yaml:21]: Line 24 here, can you change `unit_tests` to `setup` too please?

**Review by pjohnst5 — COMMENTED** [review]
AKS-swift pipeline fix

File: .pipelines/singletenancy/aks-swift/e2e-job-template.yaml (hunk 1 above)
Comment (matmerr) [reply .pipelines/singletenancy/aks-swift/e2e-job-template.yaml:21]: good catch

**Review by pjohnst5 — APPROVED** [review]
Send it 🎿

**Pull request merged**